docs/getting-started/generated-app.md

```diff
@@ -33,6 +33,9 @@ Below are the technologies employed for Node.js and .NET, ensuring efficient, re
 - [NestJS](https://nestjs.com/) - A progressive Node.js framework for building efficient, reliable and scalable server-side applications.
 - [Prisma](https://www.prisma.io/) - Next-generation ORM for Node.js and TypeScript.
 - [Passport](https://www.passportjs.org/) - Simple, unobtrusive authentication for Node.js
+- [MySQL](https://www.mysql.com/) - A popular, open source database delivering high performance scalable database applications.
```

Comment:
As discussed- let's have a database section here in which we say that we support "multiple databases, for example..." and then detail the supported databases (- 1 is missing in this list here).
But it's important to have the general statement here.